.claude/skills/run-synthetic-factory/driver.py

```
@@ -360,20 +362,22 @@ def count_nonblank_lines(path):
 
 def parse_novel_coverage(text: str):
-    """Extract 'Novel coverage: N%' from NOTES text. Returns float or None.
+    """Extract one unambiguous ``Novel coverage: N%`` from NOTES.
 
     Line-anchored parsing — matches only the labeled line (case-insensitive),
     same regex as workflow novelCoveragePct, so unrelated percentages in
-    prose never match. Valid range 0–100; out-of-range values treated as
-    unparseable to avoid false stops.
+    prose never match. Exactly one labeled line is required: duplicate lines,
+    even when numerically identical, are ambiguous audit evidence. Valid range
+    0–100; ambiguous or out-of-range values are treated as unparseable to avoid
```

Ruff (0.16.2)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)